learning_paths/models.py

```diff
@@ -185,3 +189,27 @@ def estimated_end_date(self):
         if self.learning_path.duration_in_days is None:
             return None
         return self.created + timedelta(days=self.learning_path.duration_in_days)
+
+
+class LearningPathGradingCriteria(models.Model):
+    """
+    Grading criteria for a learning path.
+
+    .. no_pii:
+    """
+
+    learning_path = models.OneToOneField(
+        LearningPath, related_name="grading_criteria", on_delete=models.CASCADE
+    )
```

Comment:
I think keeping it in a separate model is better. It will provide the flexibility to extend the grading criteria (as was initially discussed in the discovery) and allow grading-specific logic to be encapsulated.

Reply:
That's a good idea. Updated the variable names.